src/js/components/AppListComponent.jsx

```diff
@@ -178,7 +178,7 @@ var AppListComponent = React.createClass({
     var filterLabels = filters[FilterTypes.LABELS];
     var filterStatus = filters[FilterTypes.STATUS];
 
-    if (!Util.isEmptyString(filterText)) {
+    if (filterText != null && filterText !== "") {
```

Comment:
Yeah. I can't really see a use case for our `isEmptyString`, then. It should be renamed to `isStringAndNotEmpty` or something... but even so, I wonder how useful it is?